### PR TITLE
Widget filters overlapped by visualization

### PR DESCRIPTION
**Pull request by kravets-levko**
## What type of PR is this? (check all applicable)

- [x] Bug Fix

## Description

Restore missing `<div>` and add a `className` to it (with updating corresponding CSS rules) to avoid a desire to remove it next time.

### CODE REVIEW

File: client/app/visualizations/table/renderer.less [hunk 1]
```diff
@@ -103,6 +103,10 @@
 
   /* START table x scroll */
   .dashboard-widget-wrapper:not(.widget-auto-height-enabled) & {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
```
Comment (ranbena) [inline client/app/visualizations/table/renderer.less]: I'm not a big fan of this unless absolutely (pun intended) necessary.
Instead you can just add `overflow-x: auto` to:
https://github.com/getredash/redash/blob/b2c13d3d6bb473ba9969c82a132f2dc09b4304a3/client/app/pages/dashboards/dashboard.less#L69-L72

Comment (kravets-levko) [reply client/app/visualizations/table/renderer.less]: @ranbena Tried your suggestion and seems it works. Can you please briefly test it to confirm that your table scroll feature works as expected (with and without filters)? I tested it of course, but want to be sure that I didn't miss anything. Thanks!

Comment (ranbena) [reply client/app/visualizations/table/renderer.less]: Sure. I'll be able to within the hour.

Comment (kravets-levko) [reply client/app/visualizations/table/renderer.less]: Okay, this variant does not work on test environment (not sure why). I'll revert it

Comment (kravets-levko) [reply client/app/visualizations/table/renderer.less]: image

**Comment by kravets-levko** [comment]
@arikfr If you're fine with it - let's merge it

**Comment by arikfr** [comment]
:+1:

**Pull request merged**